letsencrypt/rootfs/etc/cont-init.d/file-structure.sh

```diff
@@ -6,9 +6,7 @@ mkdir -p /data/workdir
 mkdir -p /data/letsencrypt
 
 # Setup Let's encrypt config
-echo -e "dns_cloudxns_api_key = $(bashio::config 'dns.cloudxns_api_key')\n" \
-      "dns_cloudxns_secret_key = $(bashio::config 'dns.cloudxns_secret_key')\n" \
-      "dns_digitalocean_token = $(bashio::config 'dns.digitalocean_token')\n" \
+echo -e "dns_digitalocean_token = $(bashio::config 'dns.digitalocean_token')\n" \
       "certbot_dns_directadmin:directadmin_url = $(bashio::config 'dns.directadmin_url')\n" \
```

Comment:
Shouldn't this and the following also be changed from the `dist_name:plugin_name` just to `plugin_name`?

Reply:
Good catch! There are a couple of old style configs still, I've updated them all now :+1: